### PR TITLE
Carousel Component!

### PR DESCRIPTION
**Pull request by adowski**
## New Things

- Created `Carousel` component to replace ant design one in both `PropertyModal` and `PropertyView`
- Fun new arrows and images that fit the space and a click/hold to show full image feature!
- Fixed bug with property modal forms (they didn't have form names before)
- Styled modal images to align with Figma
- Got rid of unused `ApplicationModal`

### Related PRs

None

### Migrations

None

### Tests Performed, Edge Cases

None

### //TODO

Woooo

### Screenshots

image

CC: @wilsonw926

### CODE REVIEW

File: app/javascript/components/individual/Carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      images: this.props.images,
+      imgCount: this.props.images.length,
+      currentImg: 0,
+      location: this.props.location
```
Comment (wilsonw926) [inline app/javascript/components/individual/Carousel.jsx]: remove location from state

File: app/javascript/components/individual/Carousel.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    } else {
+      this.setState((state) => {
+        return {currentImg: state.imgCount - 1}
+      });
+    }
+  }
+
+
+  render() {
+    if (this.state.location === "modal") {
```
Comment (wilsonw926) [inline app/javascript/components/individual/Carousel.jsx]: convert location to props

File: app/javascript/components/individual/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+          </div>
+        );
+      } else {
+        return(
+          <div className="carousel-modal">
+            <img className="carousel-img-modal" src={this.state.images[this.state.currentImg]["url"]}/>
+          </div>
+        );
+      }
+    } else if (this.state.location === "view") {
```
Comment (wilsonw926) [inline app/javascript/components/individual/Carousel.jsx]: convert location to props

File: app/javascript/components/individual/Carousel.jsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      } else {
+        return(
+          <div className="carousel-view">
+            <img className="carousel-img-view" src={this.state.images[this.state.currentImg]["url"]}/>
+          </div>
+        );
+      }
+    }
+  }
+
```
Comment (wilsonw926) [inline app/javascript/components/individual/Carousel.jsx]: rm extra lines

File: app/javascript/components/individual/Carousel.jsx (hunk 1 above)
Comment (adowski) [reply app/javascript/components/individual/Carousel.jsx]: done

File: app/javascript/components/individual/Carousel.jsx (hunk 2 above)
Comment (adowski) [reply app/javascript/components/individual/Carousel.jsx]: done

File: app/javascript/components/individual/Carousel.jsx (hunk 3 above)
Comment (adowski) [reply app/javascript/components/individual/Carousel.jsx]: done

File: app/javascript/components/individual/Carousel.jsx (hunk 4 above)
Comment (adowski) [reply app/javascript/components/individual/Carousel.jsx]: done
